### PR TITLE
Fix WC Deck Mapping From TCG Group to Scryfall Group

### PR DESCRIPTION
**Pull request by tpblaster**
This PR adds a small filter to check for what year a certain deck is from to make sure it goes into the proper set.

### CODE REVIEW

File: mtgjson5/providers/tcgplayer.py [hunk 1]
```diff
@@ -461,11 +461,12 @@ def determine_mtgjson_sealed_product_subtype(
         return MtgjsonSealedProductSubtype.UNKNOWN
 
     def generate_mtgjson_sealed_product_objects(
-        self, group_id: Optional[int], set_code: str
+        self, group_id: Optional[int], set_code: str, wc_set: bool
```
Comment (ZeldaZach) [inline mtgjson5/providers/tcgplayer.py]: would rename to `is_wc_set`

File: mtgjson5/providers/tcgplayer.py [hunk 2]
```diff
@@ -478,6 +479,13 @@ def generate_mtgjson_sealed_product_objects(
 
         sealed_data = get_tcgplayer_sealed_data(group_id)
 
+        if wc_set:
+            correct_data = []
```
Comment (ZeldaZach) [inline mtgjson5/providers/tcgplayer.py]: `sealed_data = [product for product in sealed_data if set_code[:-2] in product["cleanName"]`

**Comment by ZeldaZach** [comment]
Reason Docker Container fails is b/c you submitted a PR from your master branch. In the future, push from a different branch.

File: mtgjson5/providers/tcgplayer.py [hunk 3]
```diff
@@ -478,6 +479,16 @@ def generate_mtgjson_sealed_product_objects(
 
         sealed_data = get_tcgplayer_sealed_data(group_id)
 
+        # adjust for worlds decks
+        if set_code in ["WC97", "WC98", "WC99", "WC00", "WC01", "WC02", "WC03", "WC04"]:
```
Comment (kodawah) [inline mtgjson5/providers/tcgplayer.py:483]: does PTC need to be here too?

File: mtgjson5/providers/tcgplayer.py [hunk 4]
```diff
@@ -478,6 +479,16 @@ def generate_mtgjson_sealed_product_objects(
 
         sealed_data = get_tcgplayer_sealed_data(group_id)
 
+        # adjust for worlds decks
```
Comment (kodawah) [inline mtgjson5/providers/tcgplayer.py:482]: id expand the comment "by looking at the last two digits being present in the product name"

File: mtgjson5/providers/tcgplayer.py [hunk 5]
```diff
@@ -478,6 +479,16 @@ def generate_mtgjson_sealed_product_objects(
...
+        if set_code in ["WC97", "WC98", "WC99", "WC00", "WC01", "WC02", "WC03", "WC04"]:
+            sealed_data = [product for product in sealed_data if set_code[:-2] in product["cleanName"]]
+
+        # adjust for mystery booster
+        if set_code == "CMB1":
```
Comment (kodawah) [inline mtgjson5/providers/tcgplayer.py:487]: does FMB1, aka the retail mb1 boxes, need special treatment too?

File: mtgjson5/set_builder.py [hunk 6]
```diff
@@ -473,6 +473,7 @@ def build_mtgjson_set(set_code: str) -> Optional[MtgjsonSetObject]:
             mtgjson_set.tcgplayer_group_id, mtgjson_set.code
         )
     )
+
```
Comment (kodawah) [inline mtgjson5/set_builder.py:476]: stray change

**Comment by kodawah** [comment]
this is unmerged, why was it closed?

**Comment by ZeldaZach** [comment]
I opened my own in favor of it @kodabb see #1054

**Pull request closed without merging**